v4/range_test.go

```
@@ -48,6 +48,9 @@ var (
 	v1 = MustParse("1.2.2")
 	v2 = MustParse("1.2.3")
 	v3 = MustParse("1.2.4")
+	v4 = MustParse("1.2.3.2")
```

golangci-lint (1.64.8)

var `v4` is unused

(unused)